server/store.go

```diff
@@ -194,3 +196,25 @@ func (p *Plugin) removeSuperUserToken() error {
 
 	return nil
 }
+
+func (p *Plugin) setFollowStatusForUser(userID string, enabled bool) *model.AppError {
+	key := fmt.Sprintf("%v_%v", zoomFollowStatusKeyByUserId, userID)
+	return p.API.KVSet(key, []byte(strconv.FormatBool(enabled)))
+}
+
+func (p *Plugin) getFollowStatusForUser(userID string) (bool, *model.AppError) {
+	key := fmt.Sprintf("%v_%v", zoomFollowStatusKeyByUserId, userID)
+	rawBool, appErr := p.API.KVGet(key)
+	if appErr != nil {
+		return false, appErr
+	}
+	if len(rawBool) == 0 {
+		return false, nil
```

Comment:
We currently don't have a way to broadcast to users that we have a new feature, so to surface new features in plugins we usually set the default to `on`. Then in a future PR (possibly during the same release as this PR) we will ask the user if they want their status changed when a meeting starts, with an option to "remember my answer". This would likely be out of scope of this PR.

```suggestion
		return true, nil
```